packages/client/src/LensClient.ts

```diff
@@ -109,6 +110,10 @@ export class LensClient {
     return new Revenue(this.config, this._authentication);
   }
 
+  get reactions(): Reactions {
```

Comment:
this doesn't make sense, Reactions are a submodule of Publication, so you access them as ie. `client.publication.reactions.add()`